.gitignore

```diff
@@ -21,6 +21,8 @@ __pycache__
 # examples related
 examples/**/package-lock.json
 examples/redteam-dalle/images
+examples/harmbench/data
```

Comment:
consider putting it right in the harmbench folder instead of harmbench/data

Reply:
I'll just keep it here in case I make additional changes to the example code. More of a frustration to manually override ignore file for specific files I want to add.